mscore/preferences.h

```diff
@@ -188,6 +188,7 @@ enum class MusicxmlExportBreaks : char {
 #define PREF_UI_PIANOROLL_DARK_NOTE_UNSEL_COLOR             "ui/pianoroll/dark/note/unselected/color"
 #define PREF_UI_PIANOROLL_DARK_NOTE_SEL_COLOR               "ui/pianoroll/dark/note/selected/color"
 #define PREF_UI_PIANOROLL_DARK_BG_BASE_COLOR                "ui/pianoroll/dark/background/base/color"
+#define PREF_UI_PIANOROLL_DARK_BG_KEY_HILIGHT_COLOR         "ui/pianoroll/dark/background/keys/hilight/color"
```

Comment:
typo? highlight?

Reply:
I've seen it spelled this way sometimes asa a short form.

Reply:
I've changed the bit that's visible to users.